app/helpers/save_this_page_helper.rb

```diff
@@ -0,0 +1,32 @@
+module SaveThisPageHelper
+  attr_reader :page_is_saved, :signed_in, :page_path
+
+  PAGE_IS_SAVED_LINK_HREF = {
+    true => "/account/delete-a-page?page_path=",
+    false => "/account/save-a-page?page_path=",
+  }.freeze
```

Comment:
We'll need to match these up with the paths in https://github.com/alphagov/frontend/pull/2781